src/Map/Map.h

```diff
@@ -43,6 +44,9 @@ class Map
     // print details of each node
     void printNodes();
 
+    // get name of starting city
```

Comment:
starting region, not city right? I guess its technically a city though xD

Reply:
Good catch, fixed.